modules/network-observability-pktdrop-overview.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-pktdrop-overview_{context}"]
+= Configuring packet drop tracking
+You can configure graphical representation of network flow records with packet loss in the *Overview* view. By employing eBPF tracepoint hooks, you can gain valuable insights into packet drops for TCP, UDP, SCTP, ICMPv4, and ICMPv6 protocols, which can result in the following actions:
+
+* Identification:: Pinpoint the exact locations and network paths where packet drops are occurring. Determine whether specific devices, interfaces, or routes are more prone to drops.
+
+* Root cause analysis:: Examine the data collected by the eBPF program to understand the causes of packet drops. For example, are they a result of congestion, buffer issues, or specific network events?
+
+* Performance optimization:: With a clearer picture of packet drops, you can take steps to optimize network performance, such as adjust buffer sizes, reconfigure routing paths, or implement Quality of Service (QoS) measures.
+
+When packet drop tracking is enabled, you can see the following metrics represented in a chart in the *Overview*. See the _Additional Resources_ of this section for more information about enabling and working with this view.
+
+* Top 5 flow dropped rates stacked
+* Total dropped rate
+* Top 5 dropped state
+* Top 5 dropped cause
+* Top 5 flow dropped rates stacked with total
```

Comment:
Can be changed to top X for all the above panels. It depends on how many the user sets, 5 is the default

Reply:
Maybe this comment was missed, its not updated still

Reply:
It looks like this was indeed missed. I addressed this in my most recent push